defender/tests.py

```diff
@@ -724,6 +729,81 @@ def test_disable_username_lockout(self):
         data_out = utils.get_blocked_usernames()
         self.assertEqual(data_out, [])
 
+    @patch('defender.config.BEHIND_REVERSE_PROXY', True)
+    @patch('defender.config.FAILURE_LIMIT', 3)
+    def test_login_blocked_for_non_standard_login_views_without_msg(self):
```

Comment:
Can you add docstrings to the test methods explaining what each test does? Not all tests have them now, and I have been meaning to fix that.